### DISCUSSION TITLE
Can I stop the app to for Accessibility Access each time app starts (though I deny it each time)?

### DISCUSSION THREAD

**Discussion by fieryWaters** [MonitorControl/MonitorControl#862] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [x] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Each time when booting the app, which is now every time my computer boots, MonitorControl asks for the Accessibility permissions which enable the keyboard brightness controls. 

While this is a nice feature, I personally don't feel comfortable giving this app so much power, and would rather use it without this feature. 

That being said, I cannot find a way to disable this annoying and repetetive "request" for permission. This app works great, but I might uninstall it if I can't get it to stop harassing me every time I boot up.

### Steps to reproduce

- Install App
- Configure to start at bootup
- Boot up computer
- Deny Accessibility Permissions
- Reboot Computer
Bug happens where it keeps annoying asking for the permissions over and over. 

### Expected behavior

There is a "Don't Ask Again" checkbox which allows me to permanantly silence the permission request. 

### Anything else?

Thank you so much for making this awesome app. 

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey 12.0.1
- Mac model: M1 Air
- MonitorControl version: 4.0.2 Build 6965
- Monitor(s): Builtin; 2nd Gen Ipad Pro 12.9"
- Apple Silicon/M1 (yes or no): YES
```

**Comment by waydabber** [comment, marked as answer]
Hi @fieryWaters, the app should ask for the permissions only if you enable media keys either for volume or brightness control - this permission is required for these functions. If you disable these, then the prompts should stop.
